src/ResourceManagement/Resource/ResourceManagement/Generated/Models/DeploymentPreFlightResource.cs

```diff
@@ -61,19 +61,5 @@ public DeploymentPreFlightResource()
         {
             this.DependsOn = new LazyList<string>();
         }
-        
-        /// <summary>
-        /// Initializes a new instance of the DeploymentPreFlightResource class
-        /// with required arguments.
-        /// </summary>
-        public DeploymentPreFlightResource(string location)
```

Comment:
@vivsriaus No, as we discussed offline, this should be handled through a customization of the partial class.